config/ad_config.py

```diff
@@ -0,0 +1,616 @@
+            "tso500": 5288,
+        },
+        "WES_TEST_STATUS": {
+            "nextseq_sequencing": 1202218804,  # Test Status = NextSEQ sequencing
+            "data_processing": 1202218805,  # Test Status = Data Processing
+        },
+    }
+    STAGE_INPUTS = {
+        "pipe": {
+            "fastqc_reads": f"-i{NEXUS_IDS['STAGES']['pipe']['fastqc']}.reads=",
+            "bwa_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['bwa']}=mem1_ssd1_v2_x8",
+            "bwa_reads1": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.reads_fastqgz=",
+            "bwa_reads2": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.reads2_fastqgz=",
+            "bwa_rg_sample": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.read_group_sample=",
+            "bwa_ref": f"-i{NEXUS_IDS['STAGES']['pipe']['bwa']}.genomeindex_targz=",
+            "picard_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['picard']}=mem1_ssd1_v2_x4",
+            "picard_bed": f"-i{NEXUS_IDS['STAGES']['pipe']['picard']}.vendor_exome_bedfile=",
+            "picard_capturetype": f"-i{NEXUS_IDS['STAGES']['pipe']['picard']}.Capture_panel=",
+            "gatk_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['gatk']}=",
+            "gatk_padding": f"-i{NEXUS_IDS['STAGES']['pipe']['gatk']}.padding=0",
+            "gatk_vcf_format": f"-i{NEXUS_IDS['STAGES']['pipe']['gatk']}.output_format=both",
+            "filter_vcf_bed": f"-i{NEXUS_IDS['STAGES']['pipe']['filter_vcf']}.bedfile=",
+            "filter_vcf_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['filter_vcf']}=mem1_ssd1_v2_x2",
+            "happy_skip": f"-i{NEXUS_IDS['STAGES']['pipe']['happy']}.skip=",
+            "happy_prefix": f"-i{NEXUS_IDS['STAGES']['pipe']['happy']}.prefix=",
+            "sambamba_instance": f"--instance-type {NEXUS_IDS['STAGES']['pipe']['sambamba']}=mem1_ssd1_v2_x2",
+            "sambamba_bed": f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.sambamba_bed=",
+            "sambamba_min_base_qual": f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.min_base_qual=",
+            "sambamba_min_mapping_qual": f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.min_mapping_qual=",
+            "sambamba_cov_level": f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.coverage_level=",
+            "sambamba_filter_cmds": (
+                f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}"
+                ".additional_filter_commands='not (unmapped or secondary_alignment)'"
+            ),
+            "sambamba_excl_dups": f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.exclude_duplicate_reads=true",
+            "sambamba_excl_failed_qual": (
+                f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.exclude_failed_quality_control=true"
+            ),
+            "sambamba_count_overl_mates": (
+                f"-i{NEXUS_IDS['STAGES']['pipe']['sambamba']}.merge_overlapping_mate_reads=true"
+            ),
+            "fhprs_skip": f"-i{NEXUS_IDS['STAGES']['pipe']['fhprs']}.skip=false",
```

Comment:
Yep - the app default is true, the fhprs command is only build for FH samples so skip=false is only specified in that case

Reply:
i assumed this was setting the default for all samples running this pipeline, so it would only need to be true for FH samples. however, I now understand that this is simply setting the dictionary, so that the relevant inputs can be used for an FH sample etc.